courses/machine_learning/deepdive2/how_google_does_ml/solutions/intro_bqml_solution.ipynb

```diff
@@ -29,18 +29,32 @@
     "---"
```

Comment:
<p>Per Lak's: Remove "More advanced" section and below (feature engineering and new model)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/616/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Section is removed.</p>